com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -378,19 +373,19 @@ public T GetOrCreateManager<T>() where T : IEOSSubManager, new()
                     manager = new T();
                     s_subManagers.Add(type, manager);
 
-                    if (manager is IEOSOnConnectLogin)
+                    if (manager is IEOSOnConnectLogin connectLogin)
                     {
-                        AddConnectLoginListener(manager as IEOSOnConnectLogin);
+                        OnConnectLogin += connectLogin.OnConnectLogin;
```

Comment:
nit: Would it make any sense to call `AddConnectLoginListener` here instead of adding directly to the event? Maybe in case in the future there's some extra code there. Probably fine as is.

Reply:
Not terribly - there isn't anything else that we do with it other than to add it to a list of delegates. This basically replaces our home-grown implementation with a system that does what we were doing much more smoothly.